tests/unittests/OperatorTest.cpp

```diff
@@ -3853,6 +3853,81 @@ TEST_P(OperatorTest, GroupConvolution) {
   EXPECT_FLOAT_EQ(result.at({0, 1, 0, 5}), (13 + 14 + 15 + 16) * 100000);
 }
 
+TEST_P(OperatorTest, GroupwiseQuantizedConvolution) {
+  ENABLED_BACKENDS(Interpreter);
```

Comment:
Shouldn't CPU/OpenCL work here since it should be lowered correctly? What about Habana?